src/hooks/useEmergencyBroadcast.ts

```
@@ -0,0 +1,87 @@
+        },
+      )
+      .subscribe();
+
+    return () => {
+      active = false;
+      supabase.removeChannel(channel);
+    };
+  }, []);
+
+  return { emergency, loading };
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)